### PR TITLE
Close #1127

### LINKED ISSUE #1127: Add S.T.A.L.K.E.R. and OpenXRay
```yaml
name: OpenXRay
originals:
- S.T.A.L.K.E.R.
type: remake
repo: 'https://github.com/OpenXRay/xray-16'
development: very active
status: playable
multiplayer:
- Online
- LAN
content: commercial
lang:
- C++
- Lua
framework:
- OpenXRay
- X-Ray
- SDL2
- DirectX
- OpenGL
- Open Dynamics Engine
license:
- BSD
info: 'Crossplatform implementation of the S.T.A.L.K.E.R. games - Windows, linux etc.'
images:
- 'https://github.com/OpenXRay/xray-16/blob/xd_dev/OpenXRayCover.png'
video:
  youtube: Alr9YQv2JWQ
```

```yaml
name: S.T.A.L.K.E.R.
names:
- Сталкер
external:
  wikipedia: S.T.A.L.K.E.R.
platform:
- Windows
meta:
  genre:
  - FPS
  subgenre:
  - Survival
  theme:
  - Alternate Historical
  - Fantasy
  - Horror
  - Modern Military
  - Post-Apocalyptic
  - Sci-Fi
```

### PR DESCRIPTION
**Pull request by myCrack**

### CODE REVIEW

**Comment by myCrack** [comment]
See also #1143.

File: games/o.yaml [hunk 1]
```diff
@@ -1800,6 +1800,36 @@
...
+  - X-Ray
+  - SDL2
+  - DirectX
+  - OpenGL
+  - Open Dynamics Engine
+  license:
+  - BSD
+  info: 'Crossplatform implementation of the S.T.A.L.K.E.R. games - Windows, linux etc.'
+  images:
+  - 'https://github.com/OpenXRay/xray-16/blob/xd_dev/OpenXRayCover.png'
```
Comment (cxong) [inline games/o.yaml:1828]: please include in-game screenshots to help visitors understand what this game looks like
also, this link should point to the raw image

File: games/o.yaml [hunk 2]
```diff
@@ -1800,6 +1800,36 @@
...
+  multiplayer:
+  - Online
+  - LAN
+  content: commercial
+  lang:
+  - C++
+  - Lua
+  framework:
+  - OpenXRay
+  - X-Ray
```
Comment (cxong) [inline games/o.yaml:1819]: remove OpenXRay and X-Ray - this field should be engines/frameworks that the game is developed using

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/o.yaml (hunk 2 above)
Comment (myCrack) [reply games/o.yaml]: Done.

**Pull request merged**